openedx/core/djangoapps/catalog/cache.py

```diff
@@ -2,4 +2,8 @@
 PROGRAM_CACHE_KEY_TPL = 'program-{uuid}'
 
 # Cache key used to locate an item containing a list of all program UUIDs.
+# This has to be deleted when removing the waffle flags populate-multitenant-programs and get-multitenant-programs
 PROGRAM_UUIDS_CACHE_KEY = 'program-uuids'
+
+# Cache key used to locate an item containing a list of all program UUIDs for a site.
+SITE_PROGRAM_UUIDS_CACHE_KEY = 'program-uuids-{site_name}'
```

Comment:
Please follow the example set above and name this `PROGRAM_UUIDS_CACHE_KEY_TPL`. The variable name should make it clear that this string is meant to be used as a template.